rllib/evaluation/rollout_worker.py

```diff
@@ -503,7 +503,7 @@ def make_env(vector_index):
                 self.async_env, self.env, self.policy_map))
 
     @DeveloperAPI
-    def sample(self):
+    def sample(self, dataset_id=None):
```

Comment:
Please don't change core APIs for adding new algorithms, instead use foreach_env / foreach_worker: https://docs.ray.io/en/master/rllib-training.html#curriculum-learning